charts/k8s-watcher/templates/metrics/deploy.yaml

```diff
@@ -28,7 +28,7 @@
 {{- end }}
 {{- end }}
 
-{{- define "metrics.daemonset.volumes" }}
+{{- define "metrics.deploy.volumes" }}
 {{- if and .Values.metrics (.Values.metrics.enabled | default false) }}
 - name: {{ include "k8s-watcher.name" . }}-metrics-config
```

Comment:
The volume name as well - it is the same of the daemonset

Reply:
nope in the daemon its -daemon-config